apps/backend/src/db/frontend/RowSchema.ts

```diff
@@ -29,6 +35,19 @@ export const CommentSchema = z.object({
 
 export type CommentSchema = z.infer<typeof CommentSchema>;
 
+export const ReportSchema = z.object({
```

Comment:
I think we should also have a UUID field in the report schema as well - it's an easy way to have a unique identifier for each comment/report. Otherwise, looks good!